manifests/integreatly-monitoring/monitoring-1.0.0/application-monitoring-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,332 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    categories: Monitoring stack
+    description: Operator managed and installs Prometheus, Alertmanager, Grafana
+    containerImage: quay.io/integreatly/application-monitoring-operator:v1.0.0-beta
+    createdAt: 2019-09-26 10:51:00 
+    support: Red Hat, Inc.
+    certified: "false"
+  name: application-monitoring-operator.v1.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+      - name: applicationmonitorings.applicationmonitoring.integreatly.org
+        version: v1alpha1
+        kind: ApplicationMonitoring
+        displayName: Integreatly monitoring stack.
+        description: Represents an installation of the Integreatly monitoring stack.
+      - name: blackboxtargets.applicationmonitoring.integreatly.org
+        version: v1alpha1
+        kind: BlackboxTarget
+        displayName: Integreatly monitoring blackbox target.
+        description: Represents a group of blackbox targets which prometheus blackbox exporter in the Integreatly monitoring stack will query.
+      - name: grafanas.integreatly.org
+        version: v1alpha1
+        kind: Grafana
+        displayName: Integreatly monitoring Grafana Operator.
+        description: Represents an installation of the Integreatly Grafana operator.
+      - name: grafanadashboards.integreatly.org
+        version: v1alpha1
+        kind: GrafanaDashboard
+        displayName: Integreatly monitoring Grafana dashboard.
+        description: Represents an instance of the Integreatly Grafana dashboard.
+      - name: grafanadatasources.integreatly.org
+        version: v1alpha1
+        kind: GrafanaDataSource
+        displayName: Integreatly monitoring Grafana datasource.
+        description: Represents an instance of the Integreatly Grafana datasource.
+  description: Placeholder description
+  displayName: Application Monitoring Operator
```

Comment:
Might need to add `replaces: application-monitoring-operator.v0.0.28`
like: https://github.com/integr8ly/manifests/blob/master/integreatly-monitoring/monitoring-0.0.28/application-monitoring-operator.v0.0.28.clusterserviceversion.yaml#L45

Reply:
hmm, good point. But will we ever upgrade from 0.0.28?

Reply:
Hmm I guess not! Just looked all the manifests in quay have been yanked.